features/TESTS/filesystem/buffered_block_device/main.cpp

```diff
@@ -23,6 +23,11 @@
 
 using namespace utest::v1;
 
+// TODO HACK, replace with available ram/heap property
+#if defined(TARGET_MTB_MTS_XDOT) || defined(TARGET_MOTE_L152RC)
```

Comment:
Excluding specific boards is probably not the best way to handle these failures.

Reply:
That is correct, TODO HACK shall be removed

Reply:
I agree that this is not ideal, but I was following precedent set here:
https://github.com/ARMmbed/mbed-os/commit/dcd0e6b82e73533ce61f69cfffd0115d94ad7a84#diff-4e5d26193a7c8cf8e05e95d7775dcf84